iwyu_path_util.cc

```diff
@@ -36,7 +37,7 @@ const char* source_extensions[] = {
   ".cxx",
   ".CXX",
   ".cpp",
-  ".CPP"
+  ".CPP",
```

Comment:
Ouch! I remember reading about this check and thinking it looked cool, but I never thought I'd put in bugs for it to find :-)